cob_linear_nav/src/cob_linear_nav.cpp

```diff
@@ -321,11 +329,33 @@ class NodeClass
 
     vec_stamped.vector = odometry->twist.twist.linear;
     vec_stamped.header.frame_id =  "base_footprint";
-    tf_listener_.transformVector(robot_frame_, vec_stamped, robot_twist_linear_robot_);
+    try
+    {
+      tf_listener_.waitForTransform(robot_frame_, vec_stamped.header.frame_id, ros::Time(0), ros::Duration(0.01));
+      tf_listener_.transformVector(robot_frame_, vec_stamped, robot_twist_linear_robot_);
```

Comment:
Can this be possible? `transformVector` has no transform time at all...
normally you should set a timestamp inside the header!?
@ipa-mig